src/textual/message.py

```diff
@@ -31,6 +31,7 @@ class Message:
         "_no_default_action",
         "_stop_propagation",
         "_handler_name",
+        "_prevent",
```

Comment:
Might be worth documenting this (and the other private attributes).

Reply:
At some point, once things settle.